greenplumpython/table.py

```diff
@@ -208,3 +231,16 @@ def values(rows: Iterable[Tuple], db: db.Database, column_names: Iterable[str] =
     rows_string = ",".join(["(" + ",".join(str(datum) for datum in row) + ")" for row in rows])
     columns_string = f"({','.join(column_names)})" if any(column_names) else ""
     return Table(f"SELECT * FROM (VALUES {rows_string}) AS vals {columns_string}", db=db)
+
+
+def order_by_str(order_by: Iterable) -> str:
```

Comment:
Wouldn't this be a private functions?

Reply:
I can make it private, the thing is that it doesn't depend on table.

Reply:
uh, ok, it is an independent function.

Since it is unlikely to be used out of the `Table` class, I suggest making it a private static function of `Table`.